tests/MEDS_Transforms/test_tokenization.py

```diff
@@ -12,8 +12,10 @@
 
 from tests.MEDS_Transforms import TOKENIZATION_SCRIPT
 
-from .test_normalization import NORMALIZED_MEDS_SCHEMA
+from ..utils import parse_meds_csvs
+from .test_normalization import NORMALIZED_MEDS_SCHEMA, WANT_HELD_OUT_0
```

Comment:
Can you combine these lines together? Or did the linter do this?

Reply:
Also, can you rename these from `WANT` to `NORMALIZED` or something during the import? Otherwise it looks like you are passing the desired outputs as inputs which is confusing.

Reply:
The linter moves these all to seperate lines, and I changed from _WANT_ to _NORMALIZED_